src/geo/ui/zoom.js

```diff
@@ -13,11 +13,11 @@ var View = require('../../core/view');
  */
 module.exports = View.extend({
 
-  className: "cartodb-zoom",
+  className: "CDB-Zoom",
 
   events: {
-    'click .zoom_in': 'zoom_in',
-    'click .zoom_out': 'zoom_out'
+    'click .js-CDB-ZoomIn': 'zoom_in',
```

Comment:
JS elements don't need to have CDB namespace, it is only for styles.
Those should be .js-zoomIn .js-zoomOut.